db/structure.sql

```diff
@@ -2122,8 +2037,6 @@ INSERT INTO schema_migrations (version) VALUES ('20160422104300');
 
 INSERT INTO schema_migrations (version) VALUES ('20160422121400');
 
-INSERT INTO schema_migrations (version) VALUES ('20160510125000');
```

Comment:
This migration file is missing from the repo. :(

Not nice to remove migration files, I think